src/client.cpp

```diff
@@ -217,6 +217,8 @@ PVR_ERROR GetAddonCapabilities(PVR_ADDON_CAPABILITIES* pCapabilities)
   pCapabilities->bHandlesInputStream         = true;
   pCapabilities->bHandlesDemuxing            = true;
   pCapabilities->bSupportsRecordingEdl       = true;
+  pCapabilities->bSupportsRecordingPlayCount = tvh->GetProtocol() >= 26 ? true : false;
+  pCapabilities->bSupportsLastPlayedPosition = tvh->GetProtocol() >= 26 ? true : false;
```

Comment:
these two lines look a little bit overcomplicated. Why not just
<pre>
 pCapabilities->bSupportsRecordingPlayCount = (tvh->GetProtocol() >= 26);
 pCapabilities->bSupportsLastPlayedPosition = (tvh->GetProtocol() >= 26);
</pre>